packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -382,15 +399,44 @@ export class ChildLogger extends TelemetryLogger {
 		if (loggerIsMonitoringContext(baseLogger)) {
 			mixinMonitoringContext(this, new CachedConfigProvider(this, baseLogger.config));
 		}
+		this.baseLogger = new FilteredLogger(baseLogger, undefined, undefined);
```

Comment:
this will create a huge perf problem with the child logger. currently child logger on child loggers expand the propeties at creation time, so we don't need to call through multiple layers of child loggers. this was done for perf reasons, as the number of object allocations was way to high

Reply:
look at line 349, this will break it

Reply:
Fixed it.